packages/hub/src/components/layout/RootLayout/index.tsx

```diff
@@ -0,0 +1,16 @@
+import { FC, PropsWithChildren } from "react";
+
+import { PageMenu } from "./PageMenu";
+import { PageFooter } from "./PageFooter";
+
+export const RootLayout: FC<PropsWithChildren> = ({ children }) => {
+  return (
+    <div className="min-h-screen flex flex-col">
+      <div className="flex-1">
+        <PageMenu />
+        <div>{children}</div>
+      </div>
+      <PageFooter />
```

Comment:
I'd expect the "isModelRoute" check to be here here, instead of in the PageFooter component.

Reply:
Oh, right. Maybe... I thought about using RSC for RootLayout, and `usePathname` is not allowed in RSC. But RSC would require a bit more refactoring in other part of the codebase, so I decided against it for now.